test/gui/shared/scripts/helpers/api/Provisioning.py

```diff
@@ -51,3 +53,36 @@ def setup_app(app_name, action):
         disable_app(app_name)
     else:
         raise Exception("Unknown action: " + action)
+
+
+def create_group(group_name):
+    url = url_join(get_ocs_url(), 'groups')
+    body = {"groupid": group_name}
+    response = request.post(url, body)
+    if response.status_code != 200:
+        raise Exception(
+            "Creating group '%s' failed with %s\n" % (group_name, response.status_code)
+            + response.text
+        )
+    # save created group
+    global created_groups
+    created_groups.append(group_name)
+
+
+def delete_groups():
+    global created_groups
+    for group_name in created_groups:
+        delete_group(group_name)
+    created_groups = []
+
+
+def delete_group(group_name):
+    global created_groups
+    url = url_join(get_ocs_url(), 'groups', group_name)
+    body = {"groupid": group_name}
+    response = request.delete(url, body)
```

Comment:
could you also add for creating group in ocis?
maybe we can have files like this:
```
ocis.py             # ocis related methods
oc10.py             # oc10 related methods
Provisionsing.py    # command and factory methods
```

Reply:
okay